package_io/constants.star

```diff
@@ -20,3 +20,8 @@ GLOBAL_CLIENT_LOG_LEVEL = struct(
 	debug="debug",
 	trace="trace",
 )
+
+GLOBAL_INDEX_ZFILL = {
+	"participants": [1, 10, 100, 1000, 10000],
```

Comment:
if this is a list of tuples then we don't have to do the zipping so - `[(1,1),  (10, 2), (100,3), (1000,4), (10000,5)]`

Reply:
is this used anywhere apart from `participant_network.star`? if not then lets move this there.

Reply:
Would probably need to use it in validators also as per https://github.com/kurtosis-tech/eth-network-package/issues/68

Reply:
we can move it then when we get to it?

Reply:
yes, I have moved it now to participant network